README.md

```diff
@@ -23,6 +26,26 @@ You can use this command in your terminal or manually create a `.env` file:
 
 `cp .env.example .env` 
 
+### PlayWright Testing Locally
+
+PlayWright is used for browser based testing. Once you have performed * [Installation via virtualenv,](docs/virtual-env.md) or you've
+got a Docker container up and running. Please perform the following below.
+
+Copy and paste into an active terminal:
+
+`playwright install`
```

Comment:
I'm following this and using a virtual env - I'm not able to execute the `playwright install` command by itself. I get this error: 

> zsh: command not found: playwright

I have to first do this:

```
pip install -r requirements/generated/requirements-dev.txt
```

I have to also make sure `FALA_URL = "http://localhost:8013/"` is in my .env file


When I do run the test using `pytest playwright` the test fails, with this error:

> FAILED playwright/e2e/test_example.py::test_check_landing_page - playwright._impl._errors.Error: Page.goto: net::ERR_CONNECTION_REFUSED at http://localhost:8013/

Reply:
I'll have to look at how you've done this as that doesn't make sense.